avocado/utils/genio.py

```diff
@@ -174,3 +174,22 @@ def write_one_line(filename, line):
     :type line: str
     """
     write_file(filename, line.rstrip('\n') + '\n')
+
+
+def safe_write_file(filename, data):
+    """
+    Write data to a file.
```

Comment:
Yes....I wanted to fail the test...but some previous comments did not recommend on error out in library code...So I thought the test should  make sure the value is set and proceed ..So is it a good thing to error out ??? or can we do something like returning  True or False